### PR TITLE
Create dotnet-desktop.yml

### LINKED ISSUE #2: Payout
https://github.com/Cordelro/payment-/blob/808a25e8ba13565a446d4bad6a82b66603cffa59/.gitignore#L5https://github.com/Cordelro/payment-/blob/808a25e8ba13565a446d4bad6a82b66603cffa59/.gitignore#L5

### PR DESCRIPTION
**Pull request by Cordelro**
#https://invoice.stripe.com/i/acct_1JQXQKlk2ucCGjAf/invst_K5k770bKQ5rQuMC

### CODE REVIEW

**Comment by Cordelro** [comment]
[https://invoice.stripe.com/i/acct_1JQXQKlk2ucCGjAf/invst_K5k770bKQ5rQuMC](stripe.com) #

>

File: .github/workflows/dotnet-desktop.yml [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+
+    # Execute all unit tests in the solution
+    - name: Execute unit tests
+      run: dotnet test
+
+    # Restore the application to populate the obj folder with RuntimeIdentifiers
+    - name: Restore the application
+      run: msbuild $env:Solution_Name /t:Restore /p:Configuration=$env:Configuration
+      env:
+        Configuration: ${{ matrix.configuration }}
```
Comment (Cordelro86) [inline .github/workflows/dotnet-desktop.yml:88]: #> > 3400
Payment

Comment (Cordelro86) [reply .github/workflows/dotnet-desktop.yml:88]: Payment confirmed

**Review by Cordelro86 — APPROVED** [review]
- - [x] Approve payment 3400

**Pull request closed without merging**